gender_analysis/analysis/metadata_visualizations.py

```diff
@@ -1,27 +1,33 @@
+from pathlib import Path
+
 import matplotlib.pyplot as plt
 import seaborn as sns
+import os
 from collections import Counter
 
 from gender_analysis.common import MissingMetadataError
 
+DEFAULT_FILE_PATH = os.getcwd() + "/gender_analysis_visualizations/"
+
 
-def plot_pubyears(corpus, filename=None):
+def plot_pubyears(corpus, file_path=DEFAULT_FILE_PATH, filename=None):
     """
-    Creates a histogram displaying the frequency of books that were published within a 20 year 
+    Creates a histogram displaying the frequency of books that were published within a 20 year
     period.
 
     *NOTE:* Requires that corpus contains a 'date' metadata field.
 
     :param corpus: Corpus object
+    :param file_path: path for where to save the file
     :param filename: Name of file to save plot as; will not write a file if None
     :return: None
 
     """
 
     if 'date' not in corpus.metadata_fields:
-        raise MissingMetadataError(['date'])
-      
-    pub_years = []
+        raise MissingMetadataError([ 'date' ])
```

Comment:
Hmm, where did these extra spaces come from? Our style has been to not add spaces inside lists like this (and throughout below). Figured our linter would scream about this but maybe not...

Reply:
My Pycharm was set to autoformat that way, so I fixed it to match the PEP8 standards that we use. Should be fixed now.